app/stylesheets/dashboard/charts-container.scss

```diff
@@ -1,5 +1,25 @@
 @import '../config/colours';
+@import '../config/dimensions';
+@import '../config/type';
 
 .charts-container {
   background: $grey-light;
+
+  .tab-label {
```

Comment:
lets make this selector more specific, since `.charts-container` will have lots of children and we don't want these rules to leak into them.
Maybe `& > .charts-container-header > .tab-label {`